### ISSUE TITLE
URI is invalid for apps

### ISSUE THREAD

**Issue by lucaponzanelli** [SeydX/homebridge-bravia-tvos#46]
The plugin fails to start up apps (e.g., Netflix). By looking at the logs, the uri of the applications seem to be correct, yet the plugin fails by saying the uri is invalid. In the old version, the same uri worked when used in `favApps`.

```
[4/30/2019, 10:04:15 PM] [BraviaOSPlatform] [INFO] Sony Bravia: Turn on Netflix                                                                                                                                                                              
2019-04-30T20:04:15.138Z BraviaPlatformApi [Bravia Debug]: api request 357: post /sony/avContent - setPlayContent - [{"uri":"kamaji://BIV-2747"}]                                                                                                            
2019-04-30T20:04:15.214Z BraviaPlatformApi [Bravia Debug]: api request 357: Error - [3,"uri is invalid"]                                                                                                                                                     
[4/30/2019, 10:04:15 PM] [BraviaOSPlatform] [ERROR] Sony Bravia: An error occured while setting new input state!                                                                                                                                             
[4/30/2019, 10:04:15 PM] [BraviaOSPlatform] [ERROR] [                                                                                                                                                                                                        
    3,                                                                                                                                                                                                                                                       
    "uri is invalid"                                                                                                                                                                                                                                         
]

**Comment by SeydX** [comment]
Ah the problem is caused to due to a wrong detection

https://github.com/SeydX/homebridge-bravia-tvos/blob/6573664716b7e1ed4797c96d75a6558143b01cee/src/accessory.js#L601

Will fix this

**Comment by SeydX** [comment]
@lucaponzanelli 

Should be fixed with v3.0.4

**Comment by lucaponzanelli** [comment]
works now, thanks!
